test/client/js/libs/handlebars/ircmessageparser/findLinks.js

```diff
@@ -103,4 +103,49 @@ describe("findLinks", () => {
 
 		expect(actual).to.deep.equal(expected);
 	});
+
+	it("does not find invalid urls", () => {
+		const input = "www.example.com ssh://-oProxyCommand=whois"; // Issue #1412
+		const expected = [{
+			start: 0,
+			end: 15,
+			link: "http://www.example.com"
+		}, {
+			end: 42,
+			start: 16,
+			link: "ssh://-oProxyCommand=whois"
+		}];
+
+		const actual = findLinks(input);
+
+		expect(actual).to.deep.equal(expected);
```

Comment:
This should be its own "should add http protocol to urls starting with www" or something, no?

Reply:
Example doesn't matter here, the test is for the second URL.

Reply:
Mmh ok